### ISSUE TITLE
Feature - Update notes.

### ISSUE THREAD

**Issue by OfficialBAMM** [dnbard/brackets-extension-rating#110]
Hello,

This isn't an issue, but I'm quite bad at finding things in GitHub, so i'll post it here.

it would be cool when there is an update, you could see the update notes. I understand that not every developer sets the releases right, or even update the text on the release, but still I think it would be a cool feature.

OfficialBAMM

**Comment by dnbard** [comment]
That is a nice feature. But there is no standart way of how developers are posting such information so it cannot be automated.

Sadly I don't see how this feature can be implemented at current stage of Brackets development.

**Comment by OfficialBAMM** [comment]
Hm shame! I thought i saw some update notes somewhere, but I understand. Thanks for the quick reply :)! Maybe in the future.

**Comment by dnbard** [comment]
Nope, let this story will be opened and maybe in future there will be instruments to implement it.

**Comment by Hirse** [comment]
I like to read the release notes of extension too and would love to be able to do so without having to search the GitHub page of the extension.

The [Brackets Extension package format](https://github.com/adobe/brackets/wiki/Extension-package-format#changelog) recommends using the proposed standard of [Keep a CHANGELOG](http://keepachangelog.com/) which includes having a file named `CHANGELOG.md` at the root level of the project.

Supporting this format might give developers an incentive to include it too. (As I already [do](https://github.com/Hirse/brackets-outline-list/blob/master/CHANGELOG.md) :wink: )

**Comment by dnbard** [comment]
:+1: Great suggestion. Will implement it soon.
